config/routes.rb

```diff
@@ -26,6 +26,10 @@
   # '/search_results'
   get '/search_results', to: 'search_results#index'
 
+  put '/users/geo_location', to: 'users/geo_locations#update_geo_location'
+
+  delete '/users/geo_location', to: 'users/geo_locations#delete_geo_location'
```

Comment:
Convention would be to call the method in the geo_locations_controller just `delete` and not `delete_geo_location`. Same with `update`